janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/ElasticsearchRunner.java

```diff
@@ -32,7 +32,7 @@
  */
 public class ElasticsearchRunner extends DaemonRunner<ElasticsearchStatus> {
 
-    private static final String DEFAULT_HOME_DIR = ".";
+    private static final String DEFAULT_HOME_DIR = System.getProperty("user.dir");
```

Comment:
why are we not pulling the index directory from GraphDatabaseConfiguration? I know the context of this default is tests, but tests use GraphDatabaseConfiguration as well.